changelogs/fragments/23-hashi-vault-lookup-refresh.yaml

```
@@ -0,0 +1,13 @@
+bugfixes:
+  - hashi_vault - when a non-token authentication method like ldap or userpass failed, but a valid token was loaded anyway (via env or token file), the token was used to attempt authentication, hiding the failure of the requested auth method.
+  - hashi_vault - if used via `with_hashi_vault` and a list of (say) 4 secrets to retrieve, only the first one would be retrieved and returned 4 times.
```

```suggestion
  - hashi_vault - if used via ``with_hashi_vault`` and a list of (say) 4 secrets to retrieve, only the first one would be retrieved and returned 4 times.
```
This is ReST, not MarkDown. (Same for other entries.)

---

Got it, changed all of them in the most recent commit. Thanks!